site/src/components/HelpTooltip/HelpTooltip.tsx

```diff
@@ -1,114 +1,78 @@
 import Link from "@mui/material/Link";
-// This is used as base for the main HelpTooltip component
-// eslint-disable-next-line no-restricted-imports -- Read above
-import Popover, { type PopoverProps } from "@mui/material/Popover";
 import HelpIcon from "@mui/icons-material/HelpOutline";
 import OpenInNewIcon from "@mui/icons-material/OpenInNew";
 import {
-  createContext,
-  useContext,
-  useRef,
-  useState,
   type FC,
   type PropsWithChildren,
   type HTMLAttributes,
   type ReactNode,
+  forwardRef,
+  ComponentProps,
 } from "react";
 import { Stack } from "components/Stack/Stack";
-import type { CSSObject } from "@emotion/css";
+import { type CSSObject } from "@emotion/css";
 import { css, type Interpolation, type Theme, useTheme } from "@emotion/react";
-import { type ClassName, useClassName } from "hooks/useClassName";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  usePopover,
+} from "components/Popover/Popover";
 
 type Icon = typeof HelpIcon;
 
 type Size = "small" | "medium";
 
-export const HelpTooltipContext = createContext<
-  { open: boolean; onClose: () => void } | undefined
->(undefined);
-
-const useHelpTooltip = () => {
-  const helpTooltipContext = useContext(HelpTooltipContext);
-
-  if (!helpTooltipContext) {
-    throw new Error(
-      "This hook should be used in side of the HelpTooltipContext.",
-    );
-  }
+export const HelpTooltipIcon = HelpIcon;
 
-  return helpTooltipContext;
+export const HelpTooltip: FC<ComponentProps<typeof Popover>> = (props) => {
+  return <Popover mode="hover" {...props} />;
 };
 
-interface HelpPopoverProps extends PopoverProps {
-  onOpen: () => void;
-  onClose: () => void;
-}
-
-export const HelpPopover: FC<HelpPopoverProps> = ({
-  onOpen,
-  onClose,
-  children,
-  ...props
-}) => {
-  const popover = useClassName(classNames.popover, []);
-  const paper = useClassName(classNames.paper, []);
+export const HelpTooltipContent = (
+  props: ComponentProps<typeof PopoverContent>,
+) => {
+  const theme = useTheme();
 
   return (
-    <Popover
-      className={popover}
-      classes={{ paper }}
-      onClose={onClose}
-      anchorOrigin={{
-        vertical: "bottom",
-        horizontal: "left",
-      }}
-      transformOrigin={{
-        vertical: "top",
-        horizontal: "left",
-      }}
-      PaperProps={{
-        onMouseEnter: onOpen,
-        onMouseLeave: onClose,
-      }}
+    <PopoverContent
       {...props}
-    >
-      {children}
-    </Popover>
+      css={{
+        "& .MuiPaper-root": {
+          fontSize: 14,
+          width: 304,
+          padding: 20,
+          color: theme.palette.text.secondary,
+        },
+      }}
+    />
   );
 };
 
-export interface HelpTooltipProps {
-  // Useful to test on storybook
-  open?: boolean;
+type HelpTooltipTriggerProps = Omit<
+  HTMLAttributes<HTMLButtonElement>,
+  "size"
```

Comment:
Idk why but when not omitting the size attribute I was getting some weird mixed type... let me check this again since the button HTML element does not have the size attribute natively.

Reply:
Fixed.